Orchid.Jint/JsExecutionContext.cs

```diff
@@ -46,7 +47,8 @@ public IJsModule NewModule(string moduleId)
         public T GetValue<T>(string name)
         {
             var value = _engine.GetValue(name);
-            return value.To<T>(_engine.ClrTypeConverter);
+            var obj = value.ToObject();  // mps TODO: This was taken from To<>. Fix to follow DRY
```

Comment:
also double check if this todo was acted upon yet or should be left

Reply:
I don't think it needs to be. Deleting the comment